packages/core/src/com/initializers/iframe.ts

```diff
@@ -18,10 +18,19 @@ export interface IIframeInitializerOptions {
      */
     hashParams?: string;
     /**
-     * if true, allows control over ifrsme content via hash parameters also.
+     * if true, allows control over iframe content via hash parameters also.
```

Comment:
I still don't understand the difference between managed and unmanaged.

What kind of control do hash params allow? And who's to stop me from changing the iframe hash params even if I don't specify `managed: true`?

Reply:
valid point. Since we do not use `managed:false` outside of engine tests and examples and there no additional reasons to keep it - can I just remove it?

Reply:
Sounds good.